README.md

```diff
@@ -2,4 +2,119 @@
+| `auth.dsn`                  | DSN connection string                                                                                                                                                                           | Required for DSN auth type.               | hdb://user:password@host443?TLSServerName=name |
+| `auth.host`                 | Sap Hana database host.                                                                                                                                                                         | Required for Basic, JWT, X509 auth types. | hdb://hanacloud.ondemand.com:443               |
+| `auth.username`             | Sap Hana user                                                                                                                                                                                   | Required for Basic type.                  | hbadmin                                        |
+| `auth.password`             | Sap Hana password                                                                                                                                                                               | Required for Basic type.                  | pass                                           |
+| `auth.token`                | JWT token                                                                                                                                                                                       | Required for JWT type.                    | jwt_token                                      |
+| `auth.clientCertFilePath`   | Path for certification file                                                                                                                                                                     | Required for X509 type.                   | /tmp/file.cert                                 |
+| `auth.ClientKeyFilePath`    | Path for key file                                                                                                                                                                               | Required for X509 type.                   | /tmp/key.cert                                  |
+
+### Snapshot
+By default when the connector starts for the first time, snapshot mode is enabled, which means that existing data will
+be read. To skip reading existing, change config parameter `snapshot` to `false`.
+
+First time when the snapshot iterator starts work,
+it is get max value from `orderingColumn` and saves this value to position.
+The snapshot iterator reads all rows, where `orderingColumn` values less or equal maxValue, from the table in batches.
```

Comment:
I'd rephrase this so that we don't mention the snapshot iterator (the iterator is an implementation detail and a user might become confused when reading this for the first time). Also, we should focus on the value of our documentation for users and how it might help them understand and use the connector. I believe the goal of this section is to explain what's considered a snapshot. If that is true, I'd suggest putting that all rows which exist in a table at the time the snapshot started, are considered part of snapshot.

Reply:
ok, I have changed it.